src/enigma2/Admin.cpp

```diff
@@ -625,18 +645,86 @@ bool Admin::GetTunerSignal(SignalStatus &signalStatus, const std::string &servic
 
   if (CanUseJsonApi())
   {
-    GetTunerDetails(signalStatus, serviceReference);
+    StreamStatus streamStatus = GetStreamDetails(channel);
+    GetTunerDetails(signalStatus, channel);
```

Comment:
This looks strange. You seem to not use `streamStatus`?

Reply:
Yes, waiting on the final API change in OpenWebIf. Will add a TODO.